lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
Hey @pimterry @emirotin cc @wrboyce , I had to made this change because `resin.models.device.getApplicationInfo(device.uuid)` is, apparently, resolving with a `String` now (instead of an `Object`), so `containerId` was set to `undefined`. 

I'm a bit confused as to why, could the latest `resin-sdk` changes be related here?